build/hack/cmd/loadtest/main.go

```diff
@@ -28,80 +27,99 @@ type BooleanRollout struct {
 }
 
 var (
-	dur       time.Duration
-	fliptAddr string
+	dur               time.Duration
+	rate              int
+	fliptAddr         string
+	fliptAuthToken    string
+	fliptCacheEnabled bool
 )
 
 func init() {
 	flag.DurationVar(&dur, "duration", 60*time.Second, "duration of load test for evaluations")
+	flag.IntVar(&rate, "rate", 100, "rate of requests per second")
 	flag.StringVar(&fliptAddr, "flipt-addr", "http://flipt:8080", "address of the flipt instance")
+	flag.StringVar(&fliptAuthToken, "flipt-auth-token", "", "token to use for authentication with flipt")
+	flag.BoolVar(&fliptCacheEnabled, "flipt-cache-enabled", false, "whether the flipt cache is enabled")
 }
 
-func run() error {
+func main() {
 	flag.Parse()
 
-	evaluationTargets := make([]vegeta.Target, 0, 2)
-
-	variantEvaluation := Evaluation{
-		EntityId: uuid.Must(uuid.NewV4()).String(),
-		FlagKey:  "flag_010",
-		Context: map[string]string{
-			"foobar": "baz",
-		},
-	}
-
-	ve, err := json.Marshal(variantEvaluation)
-	if err != nil {
-		return err
-	}
-
-	evaluationTargets = append(evaluationTargets, vegeta.Target{
-		Method: postMethod,
-		URL:    fmt.Sprintf("%s/api/v1/evaluate", fliptAddr),
-		Body:   ve,
+	evaluationTargeter := vegeta.Targeter(func(t *vegeta.Target) error {
+		t.Header = http.Header{
+			"Authorization": []string{fmt.Sprintf("Bearer %s", fliptAuthToken)},
+		}
+		t.Method = postMethod
+		t.URL = fmt.Sprintf("%s/api/v1/evaluate", fliptAddr)
```

Comment:
Are you using the legacy endpoint on purpose right here?

Reply:
updated to use the new new api